config/environments/staging.rb

```diff
@@ -31,6 +31,7 @@
 
   config.assets.compile = true
   config.assets.digest = true
+  config.assets.check_precompiled_asset = false
```

Comment:
Do we need this in `config/environments/production.rb` as well?

Reply:
yup